rma/models/rma.py

```diff
@@ -536,19 +543,18 @@ def action_rma_send(self):
 
     def action_confirm(self):
         """Invoked when 'Confirm' button in rma form view is clicked."""
-        self.ensure_one()
-        self._ensure_required_fields()
-        if self.state == 'draft':
-            if self.picking_id:
-                reception_move = self._create_receptions_from_picking()
+        for record in self.filtered(lambda r: r.state == 'draft'):
```

Comment:
Don't rewrite this, as there's no "batch work" (you do a simple loop). Just call N times from the upwards method with a single record.